src/status_im2/contexts/quo_preview/selectors/disclaimer.cljs

```diff
@@ -1,27 +1,49 @@
 (ns status-im2.contexts.quo-preview.selectors.disclaimer
   (:require [quo2.components.buttons.button :as button]
-            [quo2.components.selectors.disclaimer.view :as quo]
+            [quo2.components.selectors.disclaimer.view :as disclaimer]
             [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
             [react-native.core :as rn]
-            [reagent.core :as reagent]))
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Checked:"
+    :key   :checked?
+    :type  :boolean}
+   {:label "Blur (only for dark theme):"
+    :key   :blur?
+    :type  :boolean}])
 
 (defn cool-preview
   []
-  (let [checked? (reagent/atom false)]
+  (let [state (reagent/atom {:checked? false
+                             :blur?    true})]
     (fn []
-      [rn/view
-       {:margin-bottom      50
-        :padding-vertical   16
-        :padding-horizontal 20}
-       [rn/view
-        {:padding-vertical 60
-         :align-items      :center}
-        [quo/view
-         {:container-style {:margin-bottom 40}
-          :on-change       #(swap! checked? not)}
-         "I agree with the community rules"]
-        [button/button
-         {:disabled (not @checked?)}
+      [rn/view {:style {:flex 1}}
+       [rn/view {:style {:flex 1}}
```

Comment:
Love the usage os `:style`

Reply:
Sometimes the `:style` key is not needed, but I'm not sure about when exacly should be added or not, so I'm adding it for all the styles. :sweat_smile: 

As a side note, I did a small React Native personal project, I was compiling for both Android & browser, I noticed the styles that weren't inside the `:style` key weren't applied for browser.

Reply:
This is already on my list of things to add to the guidelines, but feel free to add if you want @ulisesmac or @ibrkhalil  :) Basically all styles should be scoped by the `:style` key, no exceptions.

Reply:
I'll add it @ilmotta :+1:

Reply:
Done in b82c0b12f65613cc3d484c6253e936eb1cd6a51b